docs/UserGuide.adoc

```diff
@@ -274,9 +274,20 @@ Examples:
 Remove a todo eatery from the todo list and provides a quick way to add it to the main list of eateries. +
 Format: `save [id]` (saves the eatery to the main list)
 
+****
+* Save the eatery at the specified `INDEX`. The index refers to the index number shown in the displayed todo list.
+* The index *must be a positive integer* 1, 2, 3, ...
+* The eatery will be removed from the todo list after `save` is executed
+* User will be required to input the missing field before the eatery is saved in the main list.
+****
+
 Examples:
 
 * `save 2`
+** Save the second eatery in the todo list
+** The eatery is removed from the todo list
+** Application will toggle to Main
+** `add` command will be constructed, allowing user to just input missing fields.
```

Comment:
I don't think you should put this here! I think you can like add this to the description of the `save` command i.e. the part right before the "Format: `save [id]`". Also, is it possible to change `[id]` in the format to `[index]` so that it's consistent !